mtp_noms_ops/apps/security/forms/check.py

```diff
@@ -112,64 +104,43 @@ def clean(self):
         """
         Makes sure that the check is in pending.
         """
+        status = self.cleaned_data['fiu_action']
+        if 'decision_reason' in self.cleaned_data:
+            reason = self.cleaned_data['decision_reason']
+        else:
+            reason = ''
+
+        if reason == '' and status == 'reject':
```

Comment:
If `self.cleaned_data['decision_reason'] = None` it would get through this check when it shouldn't. I would change the first clause of this if statement to test for truthiness rather than testing for a value of empty string. Having said that, this is probably ok as long as `decision_reason` stays a `Charfield` and no-one sets `null=True` in its constructor https://docs.djangoproject.com/en/2.0/ref/models/fields/#null

Reply:
Cheers, would `bool(reason) == False` be how you would check for Falsy values in this context? And sufficient to catch a None value?

Reply:
Just `if not reason and ...` will do nicely. `None` will evaluate to falsy so that will be covered by this